src/parsePoFile/parsePoFile.js

```diff
@@ -0,0 +1,54 @@
+const pofile = require('pofile');
```

Comment:
Can you please mind to add the GPL header and copyright to the file (some others files are missing it as well)

Reply:
Added GPL headers and copyright in 90046abfc8686f1bb6a1d1d28846cae6319bdb67